test/services/book_search_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+require_relative '../lib/fake_api.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.